### ISSUE TITLE
dash_bootstrap_components emits a deprecation warning

### ISSUE THREAD

**Issue by frnhr** [GibbsConsulting/django-plotly-dash#413]
I'm getting this warning when the server starts:
```
.../site-packages/dash_bootstrap_components/_table.py:5: UserWarning: 
The dash_html_components package is deprecated. Please replace
`import dash_html_components as html` with `from dash import html`
  import dash_html_components as html
```

This happens since Dash 2.0.

Manually updating dash_bootstrap_components to the latest version (1.2.1) removes the warning and seems to have no other effects (but I could easily be missing something). 

What is the reason for explicitly capping the version to "<1" in setup.py? 

Actually, I cannot find _any_ usages of `dash_bootstrap_components` in the source, so I'm tempted to start yelling to TEAR THIS DEPENDENCY OUT WHAT'S IT DOING HERE AAAAAA!!!!11... but then... I suspect I am probably missing a more subtle reason for this dependency, aren't I? >:]

**Comment by GibbsConsulting** [comment]
@fmhr I suspect that this is historical baggage. What happens if you forcibly upgrade the version of django-bootstrap-components? This is not a long-term fix but it would be interesting to see what happens.

**Comment by GibbsConsulting** [comment]
Now addressed in v2.1.0

**Comment by fmhr** [comment]
@frnhr
